spec/lib/que/worker_spec.rb

```diff
@@ -58,23 +106,35 @@
       end
 
       it "logs the work" do
-        ExceptionalJob.enqueue(1)
+        class ExceptionalJobWithCustomLogging < ExceptionalJob
+          custom_log_context -> (attrs) {
+            {
+              first_arg: attrs[:args][0],
+            }
+          }
+
+          @log = []
```

Comment:
What is this?

Reply:
This is something that's needed on `FakeJob` (see [here](https://github.com/gocardless/que/blob/master/spec/helpers/fake_job.rb#L4) but it's a class instance variable so it doesn't inherit